runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpInfoBox.java

```diff
@@ -56,17 +40,26 @@
 import net.runelite.client.util.LinkBrowser;
 import net.runelite.client.util.StackFormatter;
 
+import javax.swing.*;
+import javax.swing.border.EmptyBorder;
+import java.awt.*;
+import java.io.IOException;
+import java.text.DecimalFormat;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
 class XpInfoBox extends JPanel
 {
 	private static final DecimalFormat TWO_DECIMAL_FORMAT = new DecimalFormat("0.00");
 
 	// Templates
 	private static final String HTML_TOOL_TIP_TEMPLATE =
-		"<html>%s %s done<br/>"
-			+ "%s %s/hr<br/>"
-			+ "%s till goal lvl</html>";
+			"<html>%s %s done<br/>"
+					+ "%s %s/hr<br/>"
+					+ "%s till goal lvl</html>";
```

Comment:
Indentation needs to be fixed throughout all the files you've modified.

Reply:
Addressed